<launch>
	<arg name="robot_ns" default="robot0" />
	<node name="no_crash_node" type="no_crash_node" pkg="robot_no_crash" ns="$(arg robot_ns)" />
	
</launch>
